eng/Versions.props

```diff
@@ -163,6 +165,14 @@
     Upshot is we need Major.Minor.0 runtime packages for compilation and the targeting pack and Major.Minor.Latest
     runtime packages for everything else. This is not an issue for assemblies available in Microsoft.NETCore.App.Ref or
     Microsoft.Extensions.Internal.Transport because it is next to impossible we would service those packages.
+
+    System.Security.AccessControl should only be referenced in Dependencies.props and RTMVersions.csproj. Because
+    it's a transitive reference, we reship the ref/ assembly in Microsoft.AspNetCore.App.Ref. dotnet/runtime ships
+    the implementation assemblies in Microsoft.NETCore.App.Runtime.* packages.
+
+    If testing this configuration prior to servicing, update the versions of dependencies too. E.g. change
+    `$(SystemSecurityPrincipalWindowsV0PackageVersion)` if you change `$(SystemSecurityAccessControlV0PackageVersion)`
+    because System.Security.AccessControl will otherwise be loadable. This should not be necessary in servicing.
```

Comment:
@ericstj when I updated only `$(SystemSecurityAccessControlV0PackageVersion)` in testing, the _newer_ (non-RTM) copy of System.Security.AccessControl.dll ended up in our targeting pack. I believe the newer assembly was also used in most compilations. It was difficult to tell exactly why but my guess is the newer System.Security.Principal.Windows.dll was somehow considered incompatible and conflict resolution chose the newer version of both. (The System.Security.Principal.Windows package is a direct dependency of a few of our packages.)

Will we hit something similar if dotnet/runtime ships System.Security.AccessControl 5.0.1 but leaves  System.Security.Principal.Windows at 5.0.0❔ Put another way, will dotnet/runtime build but not ship a newer  System.Security.Principal.Windows.dll and use that when compiling System.Security.AccessControl.dll❔ If yes, we'll need to be _really_ careful here (perhaps adding further workarounds) and I shouldn't include the "not be necessary" words.

Reply:
There’s nothing fancy here. It just chooses the newer version. You’ll only see things go app local if you use a newer package than what’s in the shared framework you target.

Reply:
We don't generally use our own part of the shared Fx when building and the example assemblies are in Microsoft.AspNetcore.App.Ref, not Microsoft.NETCore.App.Ref. So, does "nothing fancy" help us with the corner case I describe❔

I'd like to know we can rev one without the other reliably before we get to 5.0.1. (And, these packages / assemblies have a few other inter-dependencies.)

Reply:
Unless someone objects, I'm going to assume the worst case here would be silently compiling against 5.0.x (newer) versions of these packages when servicing. Will file an issue about testing that `%(RTMVersion)` metadata is respected. That test will likely be a bit painful since we won't be building our targeting pack when servicing and checks will require looking at all references in our implementation assemblies for the relevant assemblies then checking their `[AssemblyInformationalVersion()]` attributes against the expected package version.